community-operators/service-binding-operator/0.0.24/service-binding-operator.v0.0.24-210.clusterserviceversion.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
file name should be `.....v0.0.24.cluster...`

Reply:
`service-binding-operator.v0.0.24-210.clusterserviceversion.yaml`